extensions/wikia/Chat2/Chat_setup.php

```diff
@@ -194,9 +207,35 @@
 
 		// misc
 		'chat-user-throttled',
+	],	
+	'dependencies' => [
+		'jquery.wikia',
+		'jquery.onscroll',
+		'wikia.log',
+		'wikia.cookies',
+		'wikia.history',
+		'wikia.querystring',
+		'ext.Chat2.socket.io',
+		'mediawiki.legacy.wikibits', // will make importScriptPage() available, used heavily by custom Chat.js
+		'jquery.modal', // will make $.makeModal available, used heavily by custom Chat.js
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)